src/main/java/de/muenchen/isi/domain/service/AbfrageListService.java

```diff
@@ -42,4 +43,28 @@ public AbfrageListElementsModel getAbfrageListElements() {
         abfrageListElementsModel.setListElements(listElements);
         return abfrageListElementsModel;
     }
+
+    /**
+     * Die Methode gibt {@link AbfrageListElementModel}e sortiert nach createdDateTime aufsteigend zurück.
+     *
+     * @param bauvorhabenId um Bauvorhaben zu ermitteln
+     * @return die {@link AbfrageListElementsModel}s sortiert in aufsteigender Reihenfolge.
+     */
+    public AbfrageListElementsModel getAbfrageListElementsThatReferenceBauvorhaben(final UUID bauvorhabenId) {
```

Comment:
Weshalb befindet sich diese Methode zur Extraktion der Listenelemente für die Abfragen in der Klasse AbfrageListService und die Extraktion der Listenelemente für die Infrastruktureinrichtung in der Klasse `InfrastruktureinrichtungService`?

Die Klassen AbfrageListService und BauvorahbenListService werden mit Einführung der Suche obsolet.